src/main/java/sg/ncl/testbed_interface/DataResource.java

```diff
@@ -14,5 +14,44 @@
 
     private Long id;
     private String uri;
+    private String maliciousFlag;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "maliciousFlag" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)